src/Examples/UnitModels/Operations/heat_exchanger_NTU_testing.ipynb

```diff
@@ -0,0 +1,513 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up the problem in IDAES\n",
+    "\n",
+    "First, import the required IDAES and Pyomo modules. Note that the hotside (shell) and coldside (tube) properties leverage separate property packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Import pyomo package \n",
+    "from pyomo.environ import ConcreteModel, SolverFactory, Constraint, value, units\n",
+    "\n",
+    "# Import idaes logger to set output levels\n",
+    "import idaes.logger as idaeslog\n",
+    "\n",
+    "# Import the main FlowsheetBlock from IDAES. The flowsheet block will contain the unit model\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "\n",
+    "#import the MEA property package to create a properties block for the flowsheet\n",
+    "from idaes.models_extra.column_models.properties.MEA_solvent \\\n",
+    "    import configuration as aqueous_mea\n",
+    "\n",
+    "from idaes.models.properties.iapws95 import htpx\n",
+    "\n",
+    "from idaes.models.properties.modular_properties.base.generic_property import (\n",
+    "        GenericParameterBlock)\n",
+    "\n",
+    "#Import the degrees_of_freedom function from the idaes.core.util.model_statistics package\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "\n",
+    "#Import a heat exchanger unit\n",
+    "from idaes.models.unit_models.heat_exchanger_ntu import HeatExchangerNTU as HXNTU\n",
+    "\n",
+    "#Create the ConcreteModel and the FlowsheetBlock, and attach the flowsheet block to it.\n",
+    "m = ConcreteModel()\n",
+    "\n",
+    "# Steady State Model\n",
+    "m.fs = FlowsheetBlock(dynamic=False)\n",
+    "\n",
+    "# Setup property packages for hot and cold side\n",
+    "# MEA property package for hotside\n",
+    "m.fs.hotside_properties = GenericParameterBlock(**aqueous_mea)\n",
+    "\n",
+    "# MEA property package for coldside\n",
+    "m.fs.coldside_properties = GenericParameterBlock(**aqueous_mea)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the [0D Heat Exchanger model example](http://localhost:8888/notebooks/GitHub/examples-pse/src/Examples/UnitModels/Operations/heat_exchanger_0D_testing.ipynb), the inlets are specified exactly and the solver determines heat transfer based on resulting temperatures. Here, we specify hot and cold streams and mandate that heat flow from the hot to the cold stream using the transfer unit method.\n",
```

Comment:
It isn't really clear to me what you are trying to convey here, and why the reference to the HX0D is helpful.